src/api/rpc_api/state.rs

```diff
@@ -28,37 +28,37 @@ use serde::de::DeserializeOwned;
 pub trait GetStorage<Hash> {
 	fn get_storage_value<V: Decode>(
 		&self,
-		storage_prefix: &'static str,
-		storage_key_name: &'static str,
+		pallet: &'static str,
+		key: &'static str,
 		at_block: Option<Hash>,
 	) -> Result<Option<V>>;
 
 	fn get_storage_map<K: Encode, V: Decode>(
 		&self,
-		storage_prefix: &'static str,
-		storage_key_name: &'static str,
+		pallet: &'static str,
+		key: &'static str,
 		map_key: K,
 		at_block: Option<Hash>,
 	) -> Result<Option<V>>;
 
 	fn get_storage_map_key_prefix(
 		&self,
-		storage_prefix: &'static str,
-		storage_key_name: &'static str,
+		pallet: &'static str,
+		key: &'static str,
 	) -> Result<StorageKey>;
 
 	fn get_storage_double_map<K: Encode, Q: Encode, V: Decode>(
 		&self,
-		storage_prefix: &'static str,
-		storage_key_name: &'static str,
-		first: K,
-		second: Q,
+		pallet: &'static str,
+		key: &'static str,
+		first_double_map_key: K,
+		second_double_map_key: Q,
 		at_block: Option<Hash>,
 	) -> Result<Option<V>>;
 
-	fn get_storage_by_key_hash<V: Decode>(
+	fn get_storage_by_storage_key<V: Decode>(
```

Comment:
Not sure if that's too verbose now. Maybe `get_storage_by_key` would suffice. But I'll leave the deicision to you, I don't have a hard opinion here.